execution/executor/tests/storage_integration_test.rs

```diff
@@ -106,18 +106,15 @@ fn test_genesis() {
         .unwrap();
 
     let trusted_state = TrustedState::new_trust_any_genesis_WARNING_UNSAFE();
-    let trusted_state_change = verify_update_to_latest_ledger_response(
+    verify_update_to_latest_ledger_response(
         &trusted_state,
         0,
         &request_items,
         &response_items,
         &ledger_info_with_sigs,
         &validator_change_proof,
     )
-    .unwrap();
-    if let TrustedStateChange::Stale = trusted_state_change {
-        panic!("Should never be stale since we're ratcheting from pre-genesis");
-    }
+    .expect("Should never be stale since we're ratcheting from pre-genesis");
```

Comment:
Is it proper to assume the only error that would happen is a stale response? (I mean, just unwrap()?)